apps/admin/frontend/src/screens/smart_cards_screen.tsx

```diff
@@ -0,0 +1,609 @@
+            </ImageWrapper>
+            <H2>Insert a Smart Card</H2>
+          </React.Fragment>
+        ) : (
+          <React.Fragment>
+            <ImageWrapper>
+              <RotateCardImage cardInsertionDirection="right" />
+            </ImageWrapper>
+            <H2>Card is Backwards</H2>
+          </React.Fragment>
+        )}
+      </CardIllustration>
+      <CardActions>
+        <Callout color="primary" icon="Info">
+          Insert a blank smart card to create a new card. Insert a previously
+          used card to modify it.
+        </Callout>
+        <ButtonList style={{ alignItems: 'start', marginTop: '1rem' }}>
+          <Button onPress={() => {}} disabled>
+            Create Election Manager Card
+          </Button>
+          <Button onPress={() => {}} disabled>
+            Create Poll Worker Card
+          </Button>
+          <Button onPress={() => {}} disabled>
+            Create System Administrator Card
+          </Button>
+        </ButtonList>
+      </CardActions>
+    </SmartCardsScreenContainer>
+  );
+}
+
+interface ConfirmSystemAdminCardAction {
+  actionType: 'Program' | 'PinReset';
+  doAction: VoidFunction;
+}
+
+function ConfirmSystemAdminCardActionModal({
+  actionType,
+  doAction,
+  onClose,
+}: ConfirmSystemAdminCardAction & { onClose: VoidFunction }) {
+  const { title, content, confirmLabel } = (() => {
+    switch (actionType) {
+      case 'Program':
+        return {
+          title: 'Create System Administrator Card?',
+          content: (
+            <React.Fragment>
+              <P>This card performs all system actions.</P>
+              <P>
+                Strictly limit the number created and keep all System
+                Administrator cards secure.
+              </P>
+            </React.Fragment>
+          ),
+          confirmLabel: 'Create System Administrator Card',
+        };
+
+      case 'PinReset':
+        return {
+          title: 'Reset System Administrator Card PIN?',
+          content: (
+            <P>
+              The old PIN will no longer work. After resetting the PIN, you must
+              record the new PIN to avoid being locked out of this machine.
+            </P>
+          ),
+          confirmLabel: 'Reset System Administrator Card PIN',
+        };
+
+      /* istanbul ignore next */
+      default:
+        return throwIllegalValue(actionType);
+    }
+  })();
+
+  return (
+    <Modal
+      onOverlayClick={onClose}
+      title={
+        <span>
+          <Icons.Warning color="warning" /> {title}
+        </span>
+      }
+      content={content}
+      actions={
+        <React.Fragment>
+          <Button
+            onPress={() => {
+              doAction();
+              onClose();
+            }}
+          >
+            {confirmLabel}
+          </Button>
+          <Button onPress={onClose}>Cancel</Button>
+        </React.Fragment>
+      }
+    />
+  );
+}
+
+function ProgramCard({
```

Comment:
Consider `ProgramCardInstructions`, `ProgramCardContainer`, etc. for same reasons as above